poleno/attachments/tests/test_forms.py

```diff
@@ -60,36 +60,35 @@ def _render(self, template, **context):
         return Template(template).render(Context(context))
 
 
-    @unittest.skip(u'FIXME')
     def test_new_form(self):
         form = self.AttachmentsFieldForm(attached_to=self.user1)
         rendered = self._render(u'{{ form }}', form=form)
         self.assertInHTML(u'<label for="id_attachments">Attachments:</label>', rendered)
         self.assertInHTML(u'<input id="id_attachments" name="attachments" type="hidden" value=",,">', rendered)
         self.assertInHTML(u"""
-                <span class="btn btn-default btn-file"><i class="icon-folder-open"></i> Browse
-                  <input class="fileupload" type="file" name="files" multiple="multiple" data-url="/upload/"
-                         data-field="#id_attachments" data-target="#attachments-target">
-                </span>
+                <div class="btn btn-default pln-attachments-btn">
+                  <i class="chv-icon chv-icon-lg icon-attach"></i>
+                  &nbsp;
+                  Browse
```

Comment:
@martinmacko47 Treba tiez otestovat pri roznych jazykoch? V template je `{% trans 'attachments:widget:browse' %}`.

Reply:
@viliambalaz Zatial staci defaultna anglictina. V tychto testoch netestujeme funkcnost lokalizacie, ale funkcnost fieldov a buttonov. Cize je jedno, v ktorom jazyku to je.

Reply:
@viliambalaz Neginoruje `assertInHTML` medzery? Musi byt ten HTML kod takto zalomeny na vela riadkov? Nemoze byt napriklad toto v jednom riadku v teste?
```
<i class="chv-icon chv-icon-lg icon-attach"></i>&nbsp;Browse
```

Reply:
@martinmacko47 V dokumentacii je "Whitespace in most cases is ignored ..." Nevedel som co su "most cases" :). Islo mi vsak o to, aby riadkovanie bolo rovnake ako v template `attachments_widget.html`. :ok: * medzera za `&nbsp;` je potrebna.